dev/com.ibm.ws.jca_fat_example_anno/test-resourceadapter/src/com/ibm/example/jca/anno/InteractionImpl.java

```diff
@@ -61,43 +63,49 @@ public boolean execute(InteractionSpec ispec, Record input, Record output) throw
 
         Boolean readOnly = (Boolean) con.cri.get("readOnly");
         String tableName = (String) con.cri.get("tableName");
-        ConcurrentLinkedQueue<Map<?, ?>> table = ManagedConnectionFactoryImpl.tables.get(tableName);
+        ConcurrentLinkedQueue<Map<String, String>> table = ManagedConnectionFactoryImpl.tables.get(tableName);
 
         @SuppressWarnings("unchecked")
-        Map<Object, Object> inputMap = (Map<Object, Object>) input;
+        Map<String, String> inputMap = (Map<String, String>) input;
         @SuppressWarnings("unchecked")
-        Map<Object, Object> outputMap = (Map<Object, Object>) output;
+        List<String> outputMap = (List<String>) output;
```

Comment:
Ahh, I see this is why we changed from `{}` to `[]` in our tests.  Was this change because we didn't do any IndexedRecord testing before?

Reply:
Yes, that is correct. I wanted to make sure we had coverage of both classes.